tests/src/string/src/main.sw

```diff
@@ -91,10 +111,10 @@ impl StringTest for Contract {
         assert(string.capacity() == 16);
 
         string.clear();
-        assert(string.capacity() == 16);
+        assert(string.capacity() == 0);
 
         string.push(NUMBER0);
-        assert(string.capacity() == 16);
+        assert(string.capacity() == 1);
```

Comment:
I guess these weren't working before

Reply:
No, the `clear()` function in the stdlib has changed to modify capacity.